src/main.py

```diff
@@ -1,14 +1,22 @@
 # Imports
 import discord
 import os
+from discord.enums import DefaultAvatar
 from discord.ext import commands
+import json 
 
 # Managing default and priviledged gateway intents
 intents = discord.Intents.default()
 intents.members = True
 
-# Create instance of a bot object
-client = commands.Bot(command_prefix = '-', intents = intents)
+# Function that checks for prefix for that specific guild from a set of guilds
+def get_prefix(client, message):
+    with open ("prefixes.json", "r") as f:
+        prefix = json.load(f)
+    return prefix[str(message.guild.id)]
```

Comment:
Any specific reason that we are using json?

Reply:
Yeah, we need to store the list of servers as keys and the respective prefixes as values which need to be loaded on calling the bot instance. Once the file is opened, the function returns the prefix for that server in the `commands.Bot(command_prefix = get_prefix, ...)` instance and it is done for each and every server the bot is operating in.

Reply:
Ah, yes!!